internal/kubernetes/kubernetes.go

```diff
@@ -589,3 +589,26 @@ func (k *Kubernetes) IsServiceTypeClusterIP() (bool, error) {
 
 	return service.Spec.Type == corev1.ServiceTypeClusterIP, nil
 }
+
+func (k *Kubernetes) ClusterRoles() ([]string, error) {
+	clientset, err := kubernetes.NewForConfig(k.Config)
+	if err != nil {
+		return nil, err
+	}
+
+	clusterRoles, err := clientset.RbacV1().ClusterRoles().List(context.Background(), metav1.ListOptions{})
+	if err != nil {
+		return nil, err
+	}
+
+	results := make([]string, 0, len(clusterRoles.Items))
+	for _, n := range clusterRoles.Items {
+		if strings.HasPrefix(n.Name, "system:") {
```

Comment:
Love the filtering – wondering if we should do more – i.e. perhaps anything with a colon ? or perhaps for now even an "allow" list that includes `view`, `edit`, `admin`, `cluster-admin` so we can control the experience. Alternatively we can filter on the "front-end" (i.e. ui, cli, api handlers)

Reply:
Does kubernetes set anything in `ClusterRole.ObjectMeta.Annotations` or `ClusterRole.ObjectMeta.Labels` that would identify this as a system role?

Reply:
We definitely don't want to show those cluster roles normally, but is there a use case for admins wanting to set those for a different reason?